### DISCUSSION TITLE
can possible  SSRS Reporting in ReDash ?

### DISCUSSION THREAD

**Discussion by dabhivijay2478** [getredash/redash#7168] [Q&A]
can possible  SSRS Reporting in ReDash ?

**Comment by justinclift** [comment]
By SSRS are you meaning "SQL Server Reporting Services"?

If that's the case, then... "maybe".  I'm not actually sure, but if the underlying data is available via SQL Server queries (like a normal SQL Server database) then you could *probably* create charts and dashboards using that data.

It's something you'd probably want to experiment with for a few hours before commiting to either way.
